addon/components/basic-dropdown-content.js

```diff
@@ -108,6 +108,17 @@ export default @layout(templateLayout) @tagName('')class BasicDropdownContent ex
   }
 
   // Methods
+  /**
+   * Allows similair behaviour to `ember-composable-helpers`' `optional` helper.
+   * Avoids adding extra dependencies.
+   * Can be removed when the template `V1` compatability event handlers are removed.
+   * 
+   * @see https://github.com/cibernox/ember-basic-dropdown/issues/498
+   * @memberof BasicDropdownContent
+   */
+  @action
```

Comment:
If you are not accessing `this`, there is no need of decorating the function with `@action`